commons-bom/pom.xml

```diff
@@ -37,20 +37,23 @@
     <url>https://www.wrensecurity.org</url>
 
     <properties>
+        <pgpWhitelistArtifact>org.wrensecurity:wrensec-pgp-whitelist:1.4.0</pgpWhitelistArtifact>
+
         <!-- Third party components versions -->
-        <i18nFrameworkVersion>1.4.2</i18nFrameworkVersion>
-        <jodaTime.version>2.9.7</jodaTime.version>
-        <assertj.version>2.4.1</assertj.version>
+        <i18nFrameworkVersion>1.4.3</i18nFrameworkVersion>
+        <jodaTime.version>2.10.1</jodaTime.version>
+        <assertj.version>3.11.1</assertj.version>
         <forgerock-guava.version>18.0.4</forgerock-guava.version>
-        <jackson.version>2.8.4</jackson.version>
-        <mockito.version>1.10.19</mockito.version>
-        <servlet-api.version>3.0.1</servlet-api.version>
-        <slf4j.version>1.7.21</slf4j.version>
-        <testng.version>6.9.10</testng.version>
-        <swagger.version>1.5.9</swagger.version>
+        <jackson.version>2.9.7</jackson.version>
+        <mockito.version>2.0.2-beta</mockito.version>
+        <servlet-api.version>4.0.1</servlet-api.version>
+        <slf4j.version>1.8.0-beta2</slf4j.version>
```

Comment:
Is it possible to update to non-beta version (both with SLF4J and Mockito)?

Reply:
Yes, I can do that

Reply:
That change will be in https://github.com/WrenSecurity/wrensec-commons/pull/12 because this PR is just focused on a commit coming after the updates.